web/src/components/organisms/Settings/Workspace/index.tsx

```diff
@@ -0,0 +1,198 @@
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import List from "@reearth-cms/components/atoms/List";
+import React, { useCallback, useEffect, useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+
+import useHooks, { RoleUnion } from "./hooks";
+
+const Workspace: React.FC = () => {
+  const { workspaceId } = useParams();
```

Comment:
Is there any reason you do `useParams` here instead of passing the workspaceId down from the Workspace Page as a prop?

Reply:
Will check this later when I start with layout, to check if I need to keep the current workspace param and pass it to children or capsulate the param as I did here.